src/components/Project/Project.js

```diff
@@ -191,6 +241,70 @@ const Project = ({
     [handleSetProjectData]
   )
 
+  const handleAddProjectLabel = (label, labels) => {
+    const objectLabels = {}
+
+    labels.forEach(label => {
+      const splitedLabel = label.split(':')
```

Comment:
The word split in English is also an adjective, not only a verb. Also the past tense of the verb split is split.
I suggest:
```suggestion
      const splitLabel = label.split(':')
```
Or:
```suggestion
      const labelParts = label.split(':')
```